src/js/api/order.js

```diff
@@ -301,6 +301,27 @@ ripe.Ripe.prototype.createWaybillOrderP = function(number, options) {
     });
 };
 
+ripe.Ripe.prototype.refreshShippingOrder = function(number, options, callback) {
+    callback = typeof options === "function" ? options : callback;
+    options = typeof options === "function" || options === undefined ? {} : options;
+    const url = `${this.url}orders/${number}/refresh_shipping`;
+    options = Object.assign(options, {
+        url: url,
+        method: "POST",
+        auth: true
+    });
+    options = this._build(options);
+    return this._cacheURL(options.url, options, callback);
+};
+
+ripe.Ripe.prototype.refreshShippingOrderP = function(number, options) {
+    return new Promise((resolve, reject) => {
+        this.refreshShippingOrder(number, options, (result, isValid, request) => {
+            isValid ? resolve(result) : reject(new ripe.RemoteError(request, null, result));
+        });
+    });
+};
+

```

Comment:
Add documentation similar to https://github.com/ripe-tech/ripe-sdk/blob/ns/260-bus_refresh_shipping/src/js/api/order.js#L241